source/connect/config-file.txt

```diff
@@ -0,0 +1,62 @@
+.. _config-file:
+
+==========================
+Compass Configuration File 
+==========================
+
+You can configure |compass| settings using a :manual:`configuration file 
+</reference/configuration-options>`. Configure settings with a configuration 
+file to help manage the command line options for your large-scale deployments. 
+
+Use Cases
+---------
+
+If you do not want your settings modified after startup, the configuration file 
```

Comment:
Maybe in the use cases we could call out some of the specific things you can use the configuration file for. For example, maybe we could say something like:

Use the configuration file to specify |compass-short| settings and behaviors. For example:

-  If your deployment has strict privacy requirements, you can configure |compass-short| to block outgoing network connections.
- If you need to restrict what actions users can perform in |compass-short|, you can disable the embedded shell and make |compass-short| read-only.
- If you have multiple team members using |compass-short|, you can use a common configuration file to ensure a consistent user experience.

Reply:
Done.